### PR TITLE
TS 97 2 checkout

### PR DESCRIPTION
**Pull request by john-ko**

### CODE REVIEW

File: WebContent/checkout.jsp [hunk 1]
```diff
@@ -1,22 +1,117 @@
...
+                    <h3 class="footer-header">Site Map</h3>
+                    <div><a href="ProductListingServlet?gender=women">Women</a></div>
+                    <div><a href="ProductListingServlet?gender=men">Men</a></div>
+                    <div><a href="/index.php/cart">View Cart</a></div>
+                </div>
+                <div class="third">
+                    <h3 class="footer-header">Company Info</h3>
+                    <div><a href="about">About Us</a></div>
+                    <div><a href="contact">Contact Us</a></div>
+                    <br>
```
Comment (sophiech) [inline WebContent/checkout.jsp:86]: br is used :(

File: src/tns/Checkout.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import java.util.ArrayList;
+
+import com.mysql.jdbc.PreparedStatement;
+import com.mysql.jdbc.Statement;
+
+public class Checkout extends Database {
+
+	public ArrayList<Product> getItems(String orderID) {
+		
+		//
```
Comment (sophiech) [inline src/tns/Checkout.java]: empty comment?

File: src/tns/Database.java [hunk 3]
```diff
@@ -93,16 +116,27 @@ public int save() {
...
 				throw new SQLException("Nope id's didnt return anything");
 			}
+			
+			ResultSet rs = prepared.getGeneratedKeys();
+			if (rs.next()){
+				id = rs.getInt(1);
+			} else {
+				id = 4444;
+			}
+	        setId(id);
```
Comment (sophiech) [inline src/tns/Database.java]: format weird here

File: WebContent/checkout.jsp (hunk 1 above)
Comment (john-ko) [inline WebContent/checkout.jsp:86]: SO

**Pull request merged**